helm_deploy/values-preprod.yaml

```diff
@@ -17,7 +17,7 @@ generic-service:
     API_BASE_URL_SENTENCING: https://prisoner-nomis-migration-preprod.hmpps.service.justice.gov.uk
     API_BASE_URL_ACTIVITIES: https://activities-api-preprod.prison.service.justice.gov.uk
     API_BASE_URL_NON_ASSOCIATIONS: https://non-associations-api-preprod.hmpps.service.justice.gov.uk
-    API_BASE_URL_ADJUDICATIONS: https://manage-adjudications-api-preprod.hmpps.service.justice.gov.uk
+    API_BASE_URL_ADJUDICATIONS: https://prisoner-nomis-migration-preprod.hmpps.service.justice.gov.uk
     FEATURE_EVENT_SENTENCE_ADJUSTMENT_UPSERTED: false
```

Comment:
I could change the deployment in preprod today but worried someone might do a deploy while run the performance test